server/src/main/java/org/elasticsearch/monitor/jvm/HotThreads.java

```diff
@@ -158,7 +160,9 @@ static boolean isIdleThread(ThreadInfo threadInfo) {
             if (cpuTime == INVALID_TIMING) {
                 continue;
             }
-            result.put(threadIds[i], new ThreadTimeAccumulator(threadInfos[i], cpuTime));
+            //put to result when getThreadAllocatedBytes return -1
```

Comment:
Can we please remove this comment? It doesn't seem like it's relevant anymore.

Reply:
ok 👌